source/draw/gpu/GpuCanvas.ooc

```diff
@@ -72,7 +76,7 @@ GpuCanvasYuv420Semiplanar: class extends GpuSurface {
 		this _target y canvas fill()
 		this _target uv canvas fill()
 	}
-	draw: override func ~mesh (image: Image, mesh: GpuMesh) {
+	draw: override func ~mesh (image: GpuImage, mesh: GpuMesh) {
 		yuv := image as GpuYuv420Semiplanar
```

Comment:
Ok, but in this case you are drawing to a GpuYuv420Semiplanar image, so I think the assumption is not that bad. Of course it would be nice to support any format, but that's the sort of thing we probably want to implement when it's needed.